### ISSUE TITLE
Add-LrIdentity missing -Title

### ISSUE THREAD

**Issue by TonyMasse** [LogRhythm-Tools/LogRhythm.Tools#53]
**Is your feature request related to a problem? Please describe.**
LogRhythm Identities do store the `Title` of the person. But there is no way to provide it while calling `Add-LrIdentity`

**Describe the solution you'd like**
Having a `-Title` `[string]` parameter that would allow providing the `Title` for the user/person.

**Describe alternatives you've considered**
None.

**Additional context**
Other Identities related functions might be missing this feature too.

**Comment by TonyMasse** [comment]
Actually, the code of the function is already taking care of it, but the parameter is missing.

In the `Master` branch, it's visible on line 233.
And on branch `dev-1.0.1-identity`, on line 251.

`if ($Title) {$Accounts | Add-Member -NotePropertyName title -NotePropertyValue $Title}`

It really is just a matter of adding it in the `Param()` section.

**Comment by TonyMasse** [comment]
Chaps, I made a Pull Request over branch `TonyMasse:dev-1.0.1-identity-title` (branching out directly from `LogRhythm-Tools:dev-1.0.1-identity`) with the required change. 
This version is failing when I test it, but not because of my change. So I applied the change on a another branch that doesn't have issue:
- Branch `TonyMasse:development-1.0.1-issue-53-title` (branching out of `LogRhythm-Tools:development-1.0.1`) with the same change.
